financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -0,0 +1,126 @@
+const { MedianizerPriceFeed } = require("./MedianizerPriceFeed");
+const { CryptoWatchPriceFeed } = require("./CryptoWatchPriceFeed");
+const { UniswapPriceFeed } = require("./UniswapPriceFeed");
+
+const Uniswap = require("../../core/build/contracts/Uniswap.json");
+
+async function createPriceFeed(web3, logger, networker, getTime, config) {
+  if (config.type === "cryptowatch") {
```

Comment:
This is great!

Reply:
I created an issue to implement this same logic in the other bots: https://github.com/UMAprotocol/protocol/issues/1405

Reply:
I'm adding this to liquidator/disputer bots in #1403